index.js

```diff
@@ -123,6 +123,7 @@ class Runner {
   }
 
   comment (...message) {
+    if (this.isEnded || this.isDone) throw new Error('Can\'t comment after end')
```

Comment:
you need to move this to _comment in the other class

Reply:
The above is #next and the below is 2.4.0:
![image](https://user-images.githubusercontent.com/12686176/183972192-11c14575-a022-4dd3-b9e7-e5138204c12e.png)

Is that ok?